job-dsls/jobs/downstream_pr_jobs.groovy

```diff
@@ -51,7 +51,7 @@ def final REPO_CONFIGS = [
         "optaplanner"               : [],
         "jbpm"                      : [],
         "droolsjbpm-integration"    : [
-                downstreamMvnGoals  : DEFAULTS["downstreamMvnGoals"] + ["-Pjenkins-pr-builder"]
+                downstreamMvnGoals  : DEFAULTS["downstreamMvnGoals"] + "-Pjenkins-pr-builder"
```

Comment:
@mbiarnes Please add a space in front of -Pjenkins-pr-builder as done in https://github.com/kiegroup/kie-jenkins-scripts/blob/master/job-dsls/jobs/pr_jobs.groovy#L93 .

Reply:
OK - will do this